RELEASE

```diff
@@ -1,45 +1,47 @@
 fwupd Release Notes
 
-Write release entries:
+0. Create release version
 
-git log --format="%s" --cherry-pick --right-only 1.3.2... | grep -i -v trivial | grep -v Merge | sort | uniq
+export release_ver=
+echo $release_ver > .version
```

Comment:
probably would be better to automatically create `.version` from the env variable